import-tab.js

```diff
@@ -2,8 +2,8 @@ let importItems = [];
 let existingUrls = new Set();
 
 export async function initializeImport() {
-  const { entries = [] } = await chrome.storage.local.get('entries');
-  existingUrls = new Set(entries.map(e => e.url));
+  const { snapshots = [] } = await chrome.storage.local.get('snapshots');
```

Comment:
```suggestion
  const { snapshots = [] } = await chrome.storage.local.get('snapshots');
```
careful this will drop everyone's existing data, maybe we should keep the chrome storage name as `entries` for now but call the type `Snapshot` and use the variable name `snapshots`?

Reply:
Good catch. I'll keep the storage name as `entries`. If we want, maybe we can migrate people over by following this method in `config-tab.js`:
```
  // Migrate old config_archiveboxBaseUrl to archivebox_server_url
  const {config_archiveBoxBaseUrl} = await chrome.storage.sync.get('config_archiveboxBaseUrl');
  if (config_archiveBoxBaseUrl) {
    await chrome.storage.local.set({ archivebox_server_url: config_archiveBoxBaseUrl });
  }
```

Reply:
I think just leave the storage key as `entries` for now and use `Snapshot` and `snapshots` for the JS names.